poc-step-function/lambda-functions/FetchAssetMetadata/index.js

```diff
@@ -0,0 +1,45 @@
+// This fakes reading data from S3 and just returns the expected metadata.
+
+exports.handler = async function(event, context) {
+  console.log(`Reading metadata from location ${event.AssetMetadataS3Location}.`);
+
+  /**
+   * This is an example structure for the metadata
+   */
+  return {
```

Comment:
Instead of actually retrieving this from S3, this example just spits out an example structure of the metadata.